organize/admin.py

```diff
@@ -1,14 +1,155 @@
-from django.contrib import admin
+from django.contrib import admin, messages
+from django.utils import timezone
+from django.conf.urls import url
+from django.shortcuts import redirect, get_object_or_404, render
 
 from .models import EventApplication, Coorganizer
+from .constants import ON_HOLD, IN_REVIEW, REJECTED, ACCEPTED
 
 
 class InlineCoorganizerAdmin(admin.TabularInline):
     model = Coorganizer
+    suit_classes = 'suit-tab suit-tab-organizers'
     extra = 1
 
 
+def change_status_to_on_hold(modeladmin, request, queryset):
+    queryset.update(status=ON_HOLD, status_changed_at=timezone.now())
```

Comment:
You could add a custom `change_status_to()` method on the querysets, so the API between the model and the queryset would be similar.